terraform/custom_domains/infrastructure/config/ccbl.tfvars.json

```diff
@@ -0,0 +1,34 @@
+{
+  "hosted_zone": {
+    "check-the-childrens-barred-list.education.gov.uk": {
+      "caa_records": {
+        "@": {
+          "flags": 0,
+          "tag": "issue",
+          "value": "digicert.com"
+        }
+      },
+      "txt_records": {
+        "@": {
+          "value": "v=spf1 -all"
+        },
+        "_dmarc": {
+          "value": "v=DMARC1;p=quarantine;sp=quarantine;pct=100;fo=1;rua=mailto:dmarc-rua@dmarc.service.gov.uk,mailto:dmarc-rua@education.gov.uk;ruf=mailto:DMARC.Forensic@education.gov.uk"
+        }
+      },
+      "resource_group_name": "s189p01-ccbldomains-rg",
+      "front_door_name": "s189p01-ccbl-edu-domains-fd"
+    }
+  },
+
+  "tags": {
```

Comment:
@gpeng - do these look ok?

Reply:
I think so. I'm not 100% about these is but they look the same as the others 🤷